xbmc/interfaces/legacy/Control.h

```diff
@@ -2593,7 +2585,7 @@ namespace XBMCAddon
     ///
     ///--------------------------------------------------------------------------
     /// @python_v13 New function added.
-    /// @python_v16 Deprecated **focusTexture** and **noFocusTexture**. Use **focusOnTexture** and **noFocusOnTexture**.
+    /// @python_v19 Removed **focusTexture** and **noFocusTexture**. Use **focusOnTexture** and **noFocusOnTexture**.
```

Comment:
same here. No need to keep the new api usage if it is already mentioned in the deprecated warning